servo-build-dependencies/init.sls

```diff
@@ -1,7 +1,17 @@
+{% from 'common/map.jinja' import common %}
+
 include:
   - python
 
 servo-dependencies:
+  cmd.run:
+    - name: |
+        curl https://sh.rustup.rs -sSf |
```

Comment:
This only work on Unix-like systems. On Windows we need to (somehow?) download `https://win.rustup.rs/` as `rustup-init.exe`, then run `rustup-init.exe -y --default-host x86_64-pc-windows-msvc --default-toolchain none`.